.github/workflows/ci.yml

```diff
@@ -47,7 +47,19 @@ jobs:
         with:
           version: ${{ matrix.version }}
       - uses: julia-actions/cache@v1
+
       - uses: julia-actions/julia-buildpkg@latest
+        env:
+          JULIA_PKG_PRECOMPILE_AUTO: 0
+      
+      # surface errors during precompilation
+      # auto-precompile in build step suppresses errors currently.
+      - name: Precompile
+        env:
+          JULIA_DEBUG: Plots
+        shell: julia --project=@. --color=yes {0}
+        run: |
+          using Pkg; Pkg.precompile()
```

Comment:
```suggestion
```

That does not seem to work as intended right ? No warning appear on `windows 1.8.2`.

Reply:
Yes, at the moment it does not. Was hoping for another workaround from the Pkg issue.